packages/react-components/react-card/src/components/CardHeader/useCardHeaderStyles.ts

```diff
@@ -8,46 +8,35 @@ import type { CardHeaderSlots, CardHeaderState } from './CardHeader.types';
 export const cardHeaderClassNames: SlotClassNames<CardHeaderSlots> = {
   root: 'fui-CardHeader',
   image: 'fui-CardHeader__image',
-  content: 'fui-CardHeader__content',
   header: 'fui-CardHeader__header',
   description: 'fui-CardHeader__description',
   action: 'fui-CardHeader__action',
 };
 
 const useStyles = makeStyles({
   root: {
-    display: 'flex',
-    flexDirection: 'row',
+    display: 'grid',
+    gridAutoColumns: 'min-content 1fr min-content',
+    gridAutoRows: '1fr min-content',
     alignItems: 'center',
-    ...shorthands.gap('12px'),
-    height: '32px',
   },
   image: {
-    minWidth: '24px',
-    minHeight: '24px',
-    maxWidth: '32px',
-    maxHeight: '32px',
-
-    display: 'flex',
-    alignItems: 'center',
-
-    '> *': {
-      minWidth: 'inherit',
-      minHeight: 'inherit',
-      maxWidth: '100%',
-      maxHeight: '100%',
-    },
+    marginRight: '12px',
```

Comment:
can we extract this to a js variable called `gap` or something ?

Reply:
Done 👍🏻